SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityAgentReconciliationPoller.java

```diff
@@ -54,16 +54,16 @@
 
   @Override
   public void runActionOnPoll() {
-    refereshSlavesAndRacks();
-    checkDeadSlaves();
+    refereshAgentsAndRacks();
+    checkInactiveAgents();
     inactiveAgentManager.cleanInactiveAgentsList(
       System.currentTimeMillis() -
       TimeUnit.HOURS.toMillis(configuration.getCleanInactiveHostListEveryHours())
```

Comment:
>clean up anything in InactiveAgentManager's list that is 1) being deleted by the checks you already have above or 2) older than some amount of time (likely longer than the AgentManager threshold, more like a few days)

@ssalinas should this be addressing point 2 or are they different mechanisms?

Reply:
Ah, this looks like it should be doing the 'anything old than X hours' part already